### PR TITLE
Update create-next-app template

### PR DESCRIPTION
**Pull request by ypessoa**
Updates the `create-next-app` template with a new design by @evilrabbit.

#### Updated template
new — light
new — dark

#### Previous template
old — light
old — dark

### CODE REVIEW

**Comment by ijjk** [comment]
Allow CI Workflow Run

- [ ] approve CI run for commit: 4086b1b2f74a5fdc51f762394f19fa1f292aec94

Note: this should only be enabled once the PR is ready to go and can only be enabled by a maintainer

**Comment by leerob** [comment]
The underline for "go to Next.js org" is a hover state (sorry, that could have been more clear), and should apply for all three links on the bottom 🙏

File: packages/create-next-app/templates/app-tw/js/app/page.js [hunk 1]
```diff
@@ -2,112 +2,159 @@ import Image from "next/image";
...
-      </div>
-    </main>
+      </footer>
+    </div>
+  );
+}
+
+function VercelIcon() {
+  return (
+    <svg
```
Comment (leerob) [inline packages/create-next-app/templates/app-tw/js/app/page.js:80]: Could we move these to separate files and load them with the Image tag?

**Comment by ypessoa** [comment]
> Updates the `create-next-app` template with a new design by @evilrabbit.



> The underline for "go to Next.js org" is a hover state (sorry, that could have been more clear), and should apply for all three links on the bottom 🙏

Fixed, thanks for pointing this out 👍

File: packages/create-next-app/templates/app-tw/js/app/page.js (hunk 1 above)
Comment (ypessoa) [reply packages/create-next-app/templates/app-tw/js/app/page.js]: Done ✅

**Review by samcx — APPROVED** [review]

**Review by leerob — APPROVED** [review]
Nice, thank you 💯

**Pull request merged**